### PR TITLE
Penalty system

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**

### CODE REVIEW

File: RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java [hunk 1]
```diff
@@ -174,6 +188,13 @@ public void process(GameData gameData, Entity entity) {
         float y = positionPart.getY();
         float radians = positionPart.getRadians();
         float dt = gameData.getDelta();
+        System.out.println("Penalty : " + penalty);
```
Comment (quartercastle) [inline RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java]: Remove

File: RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java [hunk 2]
```diff
@@ -174,6 +188,13 @@ public void process(GameData gameData, Entity entity) {
...
+        float actualMaxSpeed = maxSpeed / penalty;
+        System.out.println("MaxSpeed: " + maxSpeed);
```
Comment (quartercastle) [inline RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java]: Remove

**Review by quartercastle — CHANGES_REQUESTED** [review]

File: RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java (hunk 1 above)
Comment (RasmusAlbrektsen) [reply RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java]: fixed 7480e5e

File: RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java (hunk 2 above)
Comment (RasmusAlbrektsen) [reply RacingGame/Common/src/main/java/racing/common/data/entityparts/MovingPart.java]: fixed 7480e5e

**Review by quartercastle — APPROVED** [review]
👍

**Review by grammeaway — APPROVED** [review]
looks fine

**Pull request merged**